src/docs/reference/modules/Email/README.md

```diff
@@ -25,6 +25,33 @@ Enabling the `OrchardCore.Email` module will allow the user to set the following
 !!! note
     You must configure `ProxyHost` and `ProxyPort` if the SMTP server runs through a proxy server.
 
+## Email Settings Configuration
+
+The `OrchardCore.Email` module allows the user to use configuration values to override the `AdminSettings` by calling `ConfigureEmailSettings()` extension method.
```

Comment:
`AdminSettings` is not a type here (I believe you didn't mean `OrchardCore.Admin.Models.AdminSettings`). Fix the same thing in the other instances too.

So:

```suggestion
The `OrchardCore.Email` module allows the user to use configuration values to override the settings configured from the admin area by calling the `ConfigureEmailSettings()` extension method on `OrchardCoreBuilder` when initializing the app.
```